src/mut/writers.cpp

```diff
@@ -163,20 +163,26 @@ static void _write_asc_section(std::ofstream& myfile,
     }
 }
 
-void asc(const Morphology& morphology, const std::string& filename) {
+/*
+ * Returns true if the morphology satisfies requirements to be written to disk in ASC format
+ * Returns false if the morphology is empty
+ * Raises if the morphology has perimeter data
+ */
+bool _shouldBeWritten(const Morphology& morphology) {
```

Comment:
`should` -> `can`?

What's wrong with writing an empty morphology?

Reply:
The decision was made here https://github.com/BlueBrain/MorphIO/pull/128
But it's true that we could allow writing (and in this case, also reading) empty morphs. That would remove the need for this weird _shouldBeWritter function.
If we want to do this, we need to define what is a valid empty morph for each of the file formats.
Is it a totally empty file ? A file with a structure (empty dataset in the H5 case, a pair of () brackets in the ASC case ?).